custom_components/waste_collection_schedule/waste_collection_schedule/source/lumire_se.py

```diff
@@ -1,14 +1,17 @@
 from datetime import datetime
 
 import requests
+from bs4 import BeautifulSoup
 from waste_collection_schedule import Collection  # type: ignore[attr-defined]
 
 TITLE = "Luleå"
 DESCRIPTION = "Source for Luleå."
 URL = "https://www.lumire.se/"
 TEST_CASES = {
-    "Storgatan 2": {"address": "Storgatan 2"},
+    "This should fail": {"address": "Storgatan 2"},
```

Comment:
Please remove this failing test

Failing tests should suggest something is wrong. We currently don't have a way to define tests that should fail